src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'Under Construction',
+};
```

Comment:
- [ ] Kindly make sure your initial state is empty at the beginning. It must be changed only when the event is triggered to update the application status.